### PR TITLE
Added sponsors and upgraded gulp

### PR DESCRIPTION
**Pull request by felixoi**

### CODE REVIEW

**Comment by Spongy** [comment]
A preview for this pull request is available at https://spongy.github.io/SpongeHome-PRs/159/.

Here are some links to the pages that were modified:

**Comment by jamierocks** [comment]
C.c. @minecrell please review Gulp v4

File: gulpfile.js [hunk 1]
```diff
@@ -72,22 +76,27 @@ gulp.task('scss', () =>
         .pipe(gulp.dest('./dist/dev/assets/css'))
         .pipe(cleanCSS())
         .pipe(rename({suffix: '.min'}))
-        .pipe(gulp.dest('./dist/prod/assets/css'))
-);
+        .pipe(gulp.dest('./dist/prod/assets/css'));
 
-gulp.task('js', ()  =>
+    cb();
```
Comment (stephan-gh) [inline gulpfile.js]: Why do you use callbacks everywhere now? Does returning the gulp pipes no longer work?

Comment (felixoi) [reply gulpfile.js]: Actually I just read https://gulpjs.com/docs/en/getting-started/creating-tasks but as of https://gulpjs.com/docs/en/getting-started/async-completion returning streams still work. I removed the callbacks again. My bad.

**Review by stephan-gh — APPROVED** [review]
Is the tiny zap hosting logo intended? Otherwise looks good

**Comment by jamierocks** [comment]
> Is the tiny zap hosting logo intended? Otherwise looks good

Yes, sponsor logos have constraints of which to fit in - we decided that 50px would be our maximum height.

edit: Ideally, we will be supplied with a logo of better size for that location.

**Review by jamierocks — APPROVED** [review]
👍 Merging.

**Pull request closed without merging**